src/main/java/com/conveyal/r5/point_to_point/PointToPointRouterServer.java

```diff
@@ -113,6 +121,8 @@ private static void run(TransportNetwork transportNetwork) {
         ObjectReader graphQlRequestReader = mapper.reader(GraphQlRequest.class);
         ObjectReader mapReader = mapper.reader(HashMap.class);
         staticFileLocation("debug-plan");
+        transportNetwork.transitLayer.buildDistanceTables(null);
```

Comment:
Right, if you are going to do analysis work like making isochrones, you will need to build some additional tables like these. However, you have modified an existing tool that does point to point routing so that it will always build the tables when starting up. Creation of these analysis-oriented tables was intentionally made optional because they are so big and take so long to calculate. Obviously it's not a big deal on the small network you are working with, but it could take half an hour on a graph the size of the Netherlands. So if someone wants to build a graph quickly and do some routing in the Netherlands, this is making the task maybe more than twice as slow.

Currently we build the tables only in TransportNetworkCache, which is used by Analysis workers. When a network is built in that context, it contains the analysis-oriented tables which are serialized along with the graph.

When the network is built outside of an Analysis worker, those tables are not included. If you need them for your application, I think that you should have a command line switch to enable creating these extra tables. There should also be a check to see whether the deserialized graph already contains the tables, and maybe you'd want to create the tables before serialization (once) rather than after deserialization (repeatedly).